src/celeritas/field/FieldPropagator.hh

```diff
@@ -145,9 +145,6 @@ CELER_FUNCTION auto FieldPropagator<DriverT>::operator()(real_type step)
         CELER_ASSERT(substep.step <= remaining
                      || soft_equal(substep.step, remaining));
 
-        // TODO: use safety distance to reduce number of calls to
```

Comment:
Is the todo 'done' (elsewhere) or is just moot (was 'not helpful')?

Reply:
The latter: I don't think safety distance calculation will help with this particular algorithm; it certainly wouldn't have helped in this pathological case